app/javascript/components/statement.js

```diff
@@ -70,6 +70,12 @@ export default template({
       this.$nextTick( () => {
         this.$emit('search-for', field)
       })
+    },
+    scrollHere: function(event) {
+      let fragment = '#s:' + this.statement.transaction_id;
```

Comment:
Feels a bit weird that we’re reconstructing this again here, rather than pulling it out of the `href` attribute of the link we just clicked. Does Vue make the element available via `event.currentTarget` or something similar?

Reply:
Good point! It looks like we can use `event.currentTarget.hash`